sdk/include/ndk/iotypes.h

```diff
@@ -164,44 +164,49 @@ extern POBJECT_TYPE NTSYSAPI IoDriverObjectType;
 //
 // Device Node Flags
 //
-#define DNF_PROCESSED                           0x00000001
```

Comment:
Dunno if putting a comment saying that these flags are Windows 2003 compatible (or whatever other windows version) would be nice.